pom.xml

```diff
@@ -63,17 +63,8 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <github.global.server>github</github.global.server>
     <site.installationModule>google-cloud-automl-parent</site.installationModule>
-    <google.core.version>1.93.0</google.core.version>
     <google.api-common.version>1.9.2</google.api-common.version>
```

Comment:
this should be removed too right?

Reply:
There is a <link> in line 201 which use this property 
<link>https://googleapis.github.io/api-common-java/${google.api-common.version}/apidocs/</link>

Reply:
ok - let's change that to `<link>https://googleapis.github.io/api-common-java</link>` since if you type in the version the link is actually broken...